Feature - Monte Carlo Collisions with static background neutrals (#1857)
ECP-WarpX/WarpX: Docs/source/usage/parameters.rst

    This is then used in the rest of the input deck;
    in this documentation we use ``<collision_name>`` as a placeholder.

* ``<collision_name>.type`` (`string`)

>     Set to ``pairwisecoulomb`` for pairwise Coulomb collisions. This is the

Reviewer: To avoid duplication, this should say something like this:
```suggestion
    The type of collsion. The types implemented are ``pairwisecoulomb`` for pairwise Coulomb collisions and 
``background_mcc`` for collisions between particles and a neutral background. If not specified, it defaults to ``pairwisecoulomb``.
```

>     default to maintain backward compatibility.

Reviewer: ```suggestion
```

> * ``<collision_name>.species`` (`strings`, two species names separated by spaces)

Reviewer: This should describe the format for both types of collisions.

    Execute collision every # time steps.
    The default value is 1.

> A non-relativistic Monte Carlo treatment for particles colliding with a neutral,

Reviewer: ```suggestion
The ``background_mcc`` scattering type is a non-relativistic Monte Carlo treatment for particles colliding with a neutral,
```

uniform background gas is also available. The implementation follows the so-called
null collision strategy discussed for example in `Birdsall (IEEE Transactions on
Plasma Science, vol. 19, no. 2, pp. 65-85, 1991) <https://ieeexplore.ieee.org/document/106800>`_.

> * ``collisions.collision_names`` (`strings`, separated by spaces)

Reviewer: This and the following text blocks fro `type` and `species` would be removed.

    `Perez et al. (Phys. Plasmas 19, 083104, 2012) <https://doi.org/10.1063/1.4742167>`_.

* ``<collision_name>.ndt`` (`int`) optional

>     Only for ``pairwisecoulomb``. Execute collision every # time steps.

Reviewer 2: Looking at the code of this PR, it seems that `ndt` is in fact also used in `background_mcc`. Is that correct?

Author: That is correct. Thanks!

>     Only for ``pairwisecoulomb``. Execute collision every # time steps.
>     The default value is 1.

Author: ```suggestion
    Execute collision every # time steps. The default value is 1.
```
